src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/matchxpath/InputMatchXpathIllegalThrows.java

```diff
@@ -0,0 +1,13 @@
+package com.puppycrawl.tools.checkstyle.checks.coding.matchxpath;
+
+/* Config:
+ *
+ * default
```

Comment:
please update all Inputs to place there query that you used.
You used default only over one Input.

Reply:
done